cpp/src/experimental/louvain.cuh

```diff
@@ -405,67 +60,35 @@ class Louvain {
       handle_(handle),
```

Comment:
And are you copying a handle here?

Reply:
handle_ is a const reference, this is just copying the reference.

Reply:
I see.